source/solvers/fluid_dynamics_matrix_free.cc

```diff
@@ -2462,7 +2458,8 @@ FluidDynamicsMatrixFree<dim>::set_initial_condition_fd(
                level <= mg_operators.max_level();
                level++)
             {
-              mg_operators[level]->set_kinematic_viscosity(viscosity_end);
+              mg_operators[level]->properties_manager->set_rheology(
+                original_viscosity_model);
```

Comment:
You don't need to do this.
Every operator has the same rheology model since they all have a shared_ptr to the physical property manager. There is only one physical property manager so you don't need to reset the properties inside the manager
Actually, you should not even access the properties manager from inside the operator.  You should just modify it outside and in the end, everybody is pointing to the same manager.